app/services/story_updater_service.rb

```diff
@@ -8,13 +8,16 @@ def self.save(*args)
 
   def initialize(story, params = {})
     @story = story
-    @params = params
+    @params = params.to_hash
```

Comment:
TLDR; It's necessary so `attachinary` don't blow up :joy:

@marcosinger  It's a bug from the `attachinary` gem we're using, that always assume we're dealing with a `Hash`, but since it's an `ActiveSupport::HashWithIndifferentAccess` the `simbolized_keys!` is removed by rails, resulting in a 500 while performing an upload from the UI.

https://github.com/rails/rails/blob/df6048407faefdc6774a8b5a02344fd4b1a734b1/activesupport/lib/active_support/hash_with_indifferent_access.rb#L234